CHANGELOG.md

```diff
@@ -2,6 +2,137 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org).
+## Supported Release 5.0.0
+### Summary
+This is a major release which removes support for the Scientific 5 and Debian 7 OS, as well as a removal of the 'Stdlib::(Ipv4|IPv6|Ip_address)' data types in favour of 'Stdlib::IP::*'. Also contains a substantial piece of work centered around updating functions that have now been migrated into Puppet itself. Please note that this will be the last major release to support Puppet 2 and Puppet 3 and that they will soon be removed.
```

Comment:
Can you make from "Also contains" a new line, and make it bold please. Also use backticks (`) instead of (') for the Stdlib::types.

Reply:
Updated